docs/sources/InfoBlox/index.md

```diff
@@ -25,8 +25,9 @@ Warning: Despite the TA indication this data source is CIM compliant the all ver
 |----------------|----------------|----------------|----------------|
 | infoblox_dns      | infoblox:dns       | netdns          | none          |
 | infoblox_dhcp    | infoblox:dhcp      | netipam          | none          |
-| infoblox_threat    | infoblox:threat      | netids          | none          |
-| nix_syslog    | nix:syslog      | osnix          | none          |
```

Comment:
infoblox like many appliances is running linux as the base OS we don't want to propigate the linux kos across all add-ons so the nix_syslog should stay